cms/djangoapps/contentstore/views/tests/test_entrance_exam.py

```diff
@@ -140,6 +140,14 @@ def test_contentstore_views_entrance_exam_delete(self):
             resp = self.client.get(self.exam_url)
             self.assertEqual(resp.status_code, 200)
 
+            # Confirm that we have only one Entrance Exam grader after re-adding the exam (validates SOL-475)
+            graders = CourseGradingModel.fetch(self.course_key).graders
+            count = 0
+            for grader in graders:
+                if grader['type'] == 'Entrance Exam':
```

Comment:
ARG -- yes, you are right, it should be changed to use the dict as well (and there was another spot in this file, too) -- good eye, Mr. James, good eye :)